app/models/bot/alegre.rb

```diff
@@ -155,10 +155,8 @@ def self.run(body)
         if ['audio', 'image', 'video'].include?(self.get_pm_type(pm))
           self.relate_project_media_async(pm)
         else
-          Bot::Alegre.send_to_media_similarity_index(pm)
```

Comment:
Question: Why was this one deleted? Did it become redundant? Is another method doing it?